src/_gettsim_tests/test_vorsorgeaufw.py

```diff
@@ -41,6 +43,8 @@ def input_data():
 
 
 @pytest.mark.parametrize("year, target", itertools.product(YEARS, OUT_COLS))
+# testcase 19 for 2022 still fails due to missing inputs
+@pytest.mark.xfail
```

Comment:
I think the way it is implemented, all test cases are expected to fail. Instead, I propose:
add a new column to the test data `expected_to_fail`. And add a separate function `test_vorsorgeaufw_not_yet_implemented` that only uses the test cases for which `input_data["expected_to_fail"] == True` (and the existing function should only use `input_data["expected_to_fail"] == False`

Reply:
No column in the test data, please. We want to see the expected failure right next to the test, not somewhere else.

I agree that pytest syntax for expected failures with parametrize is not exactly pretty (they should really implement @tobiasraabe's syntax from pytast :wink:), but we should use that nevertheless: https://docs.pytest.org/en/latest/how-to/skipping.html#skip-xfail-with-parametrize

Reply:
Thanks for the comments! I adjusted it now and hope that the implementation is correct

Reply:
Thanks! Looks much better! If I see correctly, we, however, still do not differentiate between the test cases in 2022 which we expect to fail and those for which we do not expect that.

Hans-Martin, how would you do that if not specified in the test data? Just based on indices?

Reply:
Ah, this test structure confuses me time and again. One more reason to have them at a more granular level based on yaml.

My vote goes to keeping the current solution, i.e., ignoring some tests in 2022 that should not really be ignored.

Reply:
> My vote goes to keeping the current solution, i.e., ignoring some tests in 2022 that should not really be ignore

Then the new test cases are not run at all :/ (if I am not mistaken) Sounds not like a good solution as I am not sure when we get along to address the missing feature.

The options seem to be:
- filter failing tests by new column in data
- filter failing tests by identifiers
- remove failing tests (and copy the respective code to issue #488 )

Reply:
Ok!

Laura, can you then implement the third option?

Reply:
> Laura, can you then implement the third option?

Done!